CHANGELOG.rst

```diff
@@ -6,6 +6,7 @@ Changelog
 Current
 -------
 
+- Fix `@api.expect()` decorators with `validate=False` (:issue:`609`)
```

Comment:
It would be clearer to include that this is for when `validate=True` is set on the `Api` constructor :smile:

Reply:
Hard to formulate in one short sentence.
```suggestion
- Fix `@api.expect(..., validate=False)` decorators for an :class:`Api` where `validate=True` is set on the constructor
```
Better?